pkgs/by-name/tb/tbb_2022/package.nix

```diff
@@ -36,6 +37,13 @@ stdenv.mkDerivation (finalAttrs: {
     #
     # <https://github.com/uxlfoundation/oneTBB/pull/1849>
     ./fix-libtbbmalloc-dlopen.patch
+
+    # Only enable fcf-protection on x86 based processors
+    # <https://github.com/uxlfoundation/oneTBB/pull/1768>
+    (fetchpatch2 {
```

Comment:
`fetchpatch2` is a foot‐gun and I don’t think we should use it unless we have to (such as for renames); it’s just going to add more pain to the migration to a `fetchpatch3` now that upstream is fixing the issues we ran into with it. I realize we started recommending it in the docs though, seemingly without knowledge of its reproducibility issues, and that has already caused an unfortunate proliferation of it, so not a blocker.